### PR TITLE
Moodbar in 2.0

### PR DESCRIPTION
**Pull request by madmouser1**
Changed for Qt5 and python3 
tested on Ubuntu 17.10 with new Moodbar utility https://github.com/madmouser1/moodbar (Using Gstreamer 1.x)

### CODE REVIEW

File: plugins/moodbars/__init__.py [hunk 1]
```diff
@@ -128,13 +145,13 @@ def load(self):
...
+        self.config.setting["moodbar_vorbis_command"] = str(
             self.ui.vorbis_command.text())
-        self.config.setting["moodbar_mp3_command"] = string_(
+        self.config.setting["moodbar_mp3_command"] = str(
             self.ui.mp3_command.text())
-        self.config.setting["moodbar_flac_command"] = string_(
+        self.config.setting["moodbar_flac_command"] = str(
             self.ui.flac_command.text())
-        self.config.setting["moodbar_wav_command"] = string_(
+        self.config.setting["moodbar_wav_command"] = str(
```
Comment (sambhav) [inline plugins/moodbars/__init__.py]: Please use the string_ type to type convert to string. Its a picard internal type conversion system to handle a bunch of things.

File: plugins/moodbars/__init__.py [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
 #    'win32', 'cygwyn', 'darwin', 'os2', 'os2emx', 'riscos', 'atheos']
 
 import os.path
-from functools import partial
 from collections import defaultdict
 from subprocess import check_call
 from picard.album import Album, NatAlbum
 from picard.track import Track
 from picard.file import File
+from functools import partial
```
Comment (sambhav) [inline plugins/moodbars/__init__.py]: Any reason why this was shifted from L18 to L23? I would prefer this to be with the python standard library imports.

File: plugins/moodbars/__init__.py [hunk 3]
```diff
@@ -2,24 +2,25 @@
 
 # Changelog:
 #   [2015-09-24] Initial version with support for Ogg Vorbis, FLAC, WAV and MP3, tested MP3 and FLAC
+#   [2017-11-21] Amended to Python3 & Qt5
 
-PLUGIN_NAME = "Moodbars"
-PLUGIN_AUTHOR = "Len Joubert, Sambhav Kothari"
+PLUGIN_NAME = u"Moodbars"
+PLUGIN_AUTHOR = u"Len Joubert, Sambhav Kothari"
```
Comment (sambhav) [inline plugins/moodbars/__init__.py]: Unicode isn't present in py3. We can drop the prefix here.

**Review by sambhav — CHANGES_REQUESTED** [review]

File: plugins/moodbars/__init__.py (hunk 3 above)
Comment (madmouser1) [reply plugins/moodbars/__init__.py]: ok

File: plugins/moodbars/__init__.py (hunk 2 above)
Comment (madmouser1) [reply plugins/moodbars/__init__.py]: will do, only as I removed partial from picard.util and added straight above

**Comment by madmouser1** [comment]
many thanks @samj1912 I have amended the pull request and tested.

**Comment by madmouser1** [comment]
@samj1912 I have made the requested changes if you would like to review, many thanks

File: plugins/moodbars/__init__.py [hunk 4]
```diff
@@ -2,15 +2,17 @@
 
 # Changelog:
 #   [2015-09-24] Initial version with support for Ogg Vorbis, FLAC, WAV and MP3, tested MP3 and FLAC
+#   [2017-11-21] Amended to Python3 & Qt5
+#   [2017-11-21] removed unicode, replaced str with string_ and untrusted input on check_call addressed
```
Comment (sambhav) [inline plugins/moodbars/__init__.py]: As pointed out by codacy, can you please remove the trailing whitespace from here?

**Review by sambhav — APPROVED** [review]

**Pull request merged**